src/constants/chartConstants.ts

```diff
@@ -42,6 +42,8 @@ export const COLORS: HexColor[] = [
   '#3B3EAC',
 ];
 
+export const NEW_CHART_COLORS: HexColor[] = ['#F94144', '#F3722C', '#F8961E', '#F9C74F', '#90BE6D', '#2D9CDB'];
```

Comment:
Regarding chart colours, I don't think there are enough here - many of our pie charts have a lot of sections, so we run into repeating colours pretty quickly here.

Reply:
okie I can try something